### PR TITLE
New Janitor sprites and other stuff

### PR DESCRIPTION
**Pull request by alsoandanswer**
1. Janitorcart is now directional! (and a bunch of other janitor stuff)
1.5. You can see when water is inside buckets and mop bucket
2. Cool new reagent tanks and dispensers
3. New tank dispenser sprite

stuff

### CODE REVIEW

**Comment by flimango** [comment]
Does this sprite update retain functionality of the old janicart of seeing the signs/trash bag/mop/spray/etc inside, regardless of orientation? I think it is a good change regardless, but I am curious.

**Comment by alsoandanswer** [comment]
> Does this sprite update retain functionality of the old janicart of seeing the signs/trash bag/mop/spray/etc inside, regardless of orientation? I think it is a good change regardless, but I am curious.

It does. Adding directional sprites for them was a PITA, but worked out in the end, thankfully.

**Comment by alsoandanswer** [comment]
!review

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -187,7 +187,7 @@
 	update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bucket
-	desc = "It's a bucket."
+	desc = "This is a bucket. <i>Dear god.</i> There's more. <i>No!</i>"
```
Comment (Kaedwuff) [inline code/modules/reagents/reagent_containers/glass.dm]: What is this? More of what?  This is confusing and makes the forthright description into what I assume is some kind of hackneyed meme insertion.

Comment (alsoandanswer) [reply code/modules/reagents/reagent_containers/glass.dm]: aw man, alright i'll change it

File: code/defines/obj/weapon.dm [hunk 2]
```diff
@@ -326,6 +326,7 @@
 	throw_range = 5
 	w_class = 2.0
 	attack_verb = list("warned", "cautioned", "smashed")
+	drop_sound = 'sound/items/drop/shoes.ogg'
```
Comment (Sindorman) [inline code/defines/obj/weapon.dm]: You need to add this to the changelog

File: code/modules/reagents/reagent_containers/glass.dm [hunk 3]
```diff
@@ -198,6 +198,7 @@
 	volume = 120
 	flags = OPENCONTAINER
 	unacidable = 0
+	drop_sound = 'sound/items/drop/helm.ogg'
```
Comment (Sindorman) [inline code/modules/reagents/reagent_containers/glass.dm]: Same as above

File: code/modules/reagents/reagent_containers/glass.dm [hunk 4]
```diff
@@ -238,6 +244,7 @@ obj/item/weapon/reagent_containers/glass/bucket/wood
 	icon_state = "woodbucket"
 	item_state = "woodbucket"
 	matter = list("wood" = 50)
+	drop_sound = 'sound/items/drop/wooden.ogg'
```
Comment (Sindorman) [inline code/modules/reagents/reagent_containers/glass.dm]: Same as above

**Comment by Karolis2011** [comment]
How does reagent dispenser look on wall? Sideways on wall?

**Comment by alsoandanswer** [comment]
> How does reagent dispenser look on wall? Sideways on wall?

exactly the same as before. I just changed the sprite, I didn't touch how they attach themselves on walls.

**Pull request merged**